bundles/Eiffel/seed-data/ei_artifact/subscription.json

```diff
@@ -17,7 +17,7 @@
         {
             "conditions" : [
                 {
-                    "jmespath" : "gav.groupId=='com.othercompany.library'"
+                    "jmespath" : "data.identity=='pkg:maven/com.othercompany.library/required@required'"
```

Comment:
gav was not prefixed with 'data.' before, but identity is. Whas it wrong in the old subscription? Or should it not be prefixed now either?

Reply:
can test with data.identity. Maybe works in both way,
Only "identity" works, tried that yesterday.

Reply:
Works with "data.identity" in jmepath as well. So both ways works.

Reply:
Ok